etna/home/migrations/0011_add_feature_insights.py

```diff
@@ -0,0 +1,138 @@
+# Generated by Django 3.2.14 on 2022-08-02 21:07
```

Comment:
As discussed, multiple migrations with 0011xxx : previous fil[e 0011_add_search_image_fields.py](
https://github.com/nationalarchives/ds-wagtail/blob/develop/etna/home/migrations/0011_add_search_image_fields.py) file _etna/home/migrations/0011_add_feature_insights.py_ both ) 0011, and we want to have one single file for migrations 0011, 0012, 0013 changes in this ticket. 
Example  0012_homepage_featured_collections_and_more.py

Reply:
Got an error `django.db.utils.ProgrammingError: column "search_image_id" of relation "home_homepage" already exists`
Fix:
Restore `0011_add_search_image_fields.py`
Delete `0012xx`
Get Staging Data and run new migrations which creates new  `0012_homepage_featured_collections_and_more.py`